src/serializer/ResidualHeapSerializer.js

```diff
@@ -980,7 +980,11 @@ export class ResidualHeapSerializer {
 
   _serializeValueSymbol(val: SymbolValue): BabelNodeExpression {
     let args = [];
-    if (val.$Description) args.push(t.stringLiteral(val.$Description));
+    if (val.$Description instanceof AbstractValue) {
```

Comment:
I am surprised this already works...

Two more things:
1. In ResidualHeapVisitor, you also must visit the $Description.
2. In Emitter.js/`getReasonToWaitForDependencies`, you need to add a new case for SymbolValue where you check for $Description.